### PR TITLE
More robust and extended scripts

### PR DESCRIPTION
**Pull request by g2flyer**
* make peer a bit more robust and more complete
* handle also chaincode upgrade & chaincode list
* additional utility scripts
* some other minor cleanup

This addresses issue #111

### CODE REVIEW

File: fabric/bin/peer.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -vx
```
Comment (mbrandenburger) [inline fabric/bin/peer.sh]: Do you really wanna keep this `-vx`? It spams my terminal.

Comment (mbrandenburger) [reply fabric/bin/peer.sh]: Can here also please add a short title to the commit message. Thank you

Comment (g2flyer) [reply fabric/bin/peer.sh]: removed -vx (leftover from debugging) and added short title to commit list

Comment (mbrandenburger) [reply fabric/bin/peer.sh]: thanks

**Comment by mbrandenburger** [comment]
@bvavala please review and merge if no changes are needed. Thanks

**Pull request merged**